autolab/Autolab
# Add correct timestamps to contents of zip file and so on

## Description
- Add correct timestamps to contents of zip file
- More comprehensible downloaded file name
- Removal of intermediate unnecessary files

## Motivation and Context
A rebase of #1099 due to the code changes since then.

## How Has This Been Tested?



1. Verify that temporary files are generated for downloading all submissions on the current master branch. The file position with respect to the machine system can be found in the rails log.
2. Checkout this branch and download all submissions again. The downloaded file should have a more comprehensible name. No temporary file with the same name is generated anywhere on the computer.

## Types of changes


## Checklist:


## Other issues / help required




If unsure, feel free to submit first and we'll help you along.
